utkdigitalinitiatives/utk_isl_xml_forms
# Con 36

**JIRA Ticket**: [CON-36](https://jira.lib.utk.edu/browse/CON-36)

# What does this Pull Request do?

This request will add a new Islandora form that will streamline the creation of new records for the Smoky Mountains Hiking Club collection. Here's an existing [record](https://digital.lib.utk.edu/collections/islandora/object/smhc%3A11317/datastream/MODS/view). Note that edtf dates will be dealt with in a post-process transform.

# What's new?
A new form has been added.

# How should this be tested?

- Start up your Vagrant
- Import the new form
- Associate it with your favorite content model
- Create a new object of your favorite content model and select the newly associated form
- Now edit the record and make sure it reads the values in correctly
- Now open the MODS and look at the record. Does all look good? Are there blank nodes? Are any duplicate nodes written into the XML upon updating?

# Interested parties
@markpbaggett
